### PR TITLE
Fixing appearance of Not-A-Number values when gradient is 0

### PR DESCRIPTION
**Pull request by dzenanz**
The fix could have been a bit more elegant of Normalize() returned the previous norm.

### CODE REVIEW

**Comment by thewtex** [comment]
I will update the Docker image ITK version in another branch, then we can rebase this one.

**Comment by thewtex** [comment]
Nice!

File: include/itkCuberilleImageToMeshFilter.hxx [hunk 1]
```diff
@@ -450,6 +450,10 @@ CuberilleImageToMeshFilter<TInputImage,TOutputMesh,TInterpolator>
     {
     // Compute normal vector
     normal = m_GradientInterpolator->Evaluate( vertex );
+    if (normal.GetSquaredNorm() == 0)
```
Comment (hjmjohnson) [inline include/itkCuberilleImageToMeshFilter.hxx]: Should this be "< epsilon" value?  I'm concerned if 1e-96 is an OK value?

Comment (dzenanz) [inline include/itkCuberilleImageToMeshFilter.hxx]: This problem (gradient==0) appears when the input image is a binary mask. If norm is non-zero (even 1e-96) that will lead to a non-NaN vector which is fine.

**Pull request merged**